Apps/AdminWebClient/src/ClientApp/src/views/Feedback.vue

```diff
@@ -151,6 +183,25 @@ export default class FeedbackView extends Vue {
         return this.adminTags.map<string>((x) => x.name);
     }
 
+    private get filterIsActive(): boolean {
+        return this.selectedAdminTagIds.length > 0;
+    }
+
+    private get filteredFeedbackList(): UserFeedback[] {
```

Comment:
this function is just a wrapper around ```includes``` method. Lets use includes directly on the logic that needs it